doc/yeast-capture.md

```diff
@@ -0,0 +1,77 @@
+
+The colonies on plate are observed as circles, often isolated, indicating effective single-cell distribution.
+
+TODO: analyze plate photographs to obtain some information of colonies distribution
+
+TODO: estimate how well we could identify overlapping circles
+
+TODO: literature reference on this
+
+PRIOR
+Worst case estimate encountered in [Yeast: White, Zainacheff](ISBN-13: 978-0937381960) seems to be around 20% probability of pickup of colony from more than one origin cell. We will use worst-case for now.
+
+Thus, plating from culture with probability of population being monoclonal Ps, we produce population with probability of being monoclonal Pf = Ps + (1-Ps) * 0.8
+
+PRIOR
+With worst-case scenario, Ps = 0 (capture from multiclonal source), Pf0 = 0.8
+
+Just repeated streaking operations result in:
+
+Pf1 = 0.8 + 0.2 * 0.8 = 0.96
+
+Pf2 = 0.96 + 0.04 * 0.8 = 0.992
+
+Having 99% probability of pure culture is our initial goal; process with 3 plating stages promises to result in this value if no other significant sources of contamination are introduced.
+
+If divergence is observed on some stage, we should identify that stage as first streaking stage, as Ps = 1.
+
+## Grow stages
+
+Colonies picked up from plates are grown in tubes before new platings. This stage places additional selective pressure on non-monoclonal populations, but also intoduces probability of mutations.
+
+PRIOR
+TODO: Probability to contaminate test tube could be estimated as negligible for now (P = 0), as no events were identified yet.
+
+PRIOR
+TODO: Probability of mutation should be calculated from fission rate and mutation rate, that should be yet found. This happens occasionally, something that looks like petite mutants was observed in numerous experiments, especially where yeast that participated in long fermentations or storage was propagated.
+
+## Population-based approach
+
+Instead of computing probability of having pure culture, which makes error assessment difficult if possible at all, we should probaby use fraction of pedigree yeast in carrier as parameter.
+
+It does not really matter what the distribution is in starting material on catch stage; any yeast can be chosen as "target" and we do not really care which at this point.
+
+PRIOR
+On plating stage, we can assume zero-truncated (since we definitely observe a colony) Poisson distribution of number of strains in a colony, with probability of 1 colony at 80%. This results in zero-truncated Poisson parameters:
+
+P(1) = 0.8
+0.8 * (e^l - 1) = l
+l = 0.4
+
+This is approximate value giving 81% probability of 1-colony pickup; let's re-define it as actual prior, since underlying value was also given with less than 1 significant figure.
+
+Thus, after first pick-up, fraction of good cells in a tube is 1 with 81% probability, if it's 2 population, then fraction of good cells is uniformly distributed between 0.5 and 1 (since if it's less than 0.5, we name the other one "good"). And so forth.
+
```

Comment:
Let's use a common markdown style for links.

[^White-Zainacheff]: Yeast, The Practical Guide to Beer Fermentation. ISBN-13: 978-0937381960.